crd-generator/test/src/test/java/io/fabric8/crd/generator/approvaltests/annotated/AnnotatedSpec.java

```diff
@@ -42,6 +42,15 @@ public class AnnotatedSpec {
   private String defaultValue;
   @Default("my-value2")
   private String defaultValue2;
+  @JsonProperty(defaultValue = "true")
```

Comment:
There's a precedence test for this, you can check it out ;) 
JsonProperty takes precedence since AFAIR the other annotation will eventually be removed or deprecated.

Reply:
> AFAIR the other annotation will eventually be removed or deprecated

oh, really? I'm surprised!

Should this behavior be documented? [somewhere](https://github.com/fabric8io/kubernetes-client/blob/main/doc/CRD-generator.md#iofabric8generatorannotationdefault)

Reply:
I have really no idea, I deduced this from this `TODO` comment:

https://github.com/fabric8io/kubernetes-client/blob/bee2f0154aa088a0343e2a045766c6ed91701e28/crd-generator/api-v2/src/main/java/io/fabric8/crdv2/generator/AbstractJsonSchema.java#L281

I thought this had been discussed, but maybe not.

TBH this is the first time I'm actually looking into the CRD generator, up until now I had been delegating any task related to the CRD generator to you folks. So I might be disconnected.

In any case, a deprecation would happen in 7.x and not in 7.0.0